### ISSUE TITLE
SQLite.Interop.dll not found in Win32 project

### ISSUE THREAD

**Issue by ptsoccer** [oysteinkrog/SQLite.Net-PCL#350]
Seems similar to https://github.com/oysteinkrog/SQLite.Net-PCL/issues/176, but not exactly. In a vanilla install in a Win32 project, SQLite.Interop.dll is not included in the nuget package causing an error to be thrown saying the dll isn't found. I had to manually add a reference to System.Data.SQLite.Core, which is fine, however the documentation on the main page says that the Win32 platform "bundles sqlite binaries for windows, works on both x86 and x64 automatically".

**Comment by djpflager** [comment]
I was also confused by this.

**Comment by PandaL33** [comment]
You should copy X86 and x64 to the directory of your project.

**Comment by ptsoccer** [comment]
??? I'm not sure if you fully read the issue or not, but the problem is that the nuget package does not contain the native interop DLL at all. Even if that was the case, I would not expect the documentation to say that it works automatically when you have to copy the file manually.

**Comment by PandaL33** [comment]
You're right, I solve the problem by copying the X86 and x64 to the directory of my project.

**Comment by PandaL33** [comment]
X86 and x64 folders contain the native interop DLL, X86 and x64 folders are in the “SQLite.Net-PCL/src/SQLite.Net.Platform.Win32/”

**Comment by ptsoccer** [comment]
Thanks, but as I said in the initial post, I already got it working by adding a reference to the System.Data.SQLite.Core nuget package. My point in making this issue is not to get my program working as I have already have it working, but to point out that either the documentation is wrong, or the nuget package is wrong.

**Comment by riyadparvez** [comment]
I've faced the same issue. Is there any fix for this?

**Comment by hidekuro** [comment]
I got same issue. Do I have to install the System.Data.SQLite.Core package?
